frontend/src/v5/ui/controls/dueDate/dueDateWithLabel/dueDateWithLabel.component.tsx

```diff
@@ -0,0 +1,43 @@
+/**
+ *  Copyright (C) 2023 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { BaseDueDate } from '../baseDueDate.component';
+import { DueDateEmptyLabel } from './dueDateLabel/dueDateEmptyLabel.component';
+import { DueDateFilledLabel } from './dueDateLabel/dueDateFilledLabel.component';
+
+type IDueDateWithLabel = {
+	value: number;
+	disabled?: boolean;
+	onBlur?: (newValue) => void;
+};
+
+export const DueDateWithLabel = ({ value, disabled, onBlur }: IDueDateWithLabel) => (
+	<BaseDueDate
+		value={value}
+		disabled={disabled}
+		onBlur={onBlur}
+		renderInput={({ inputRef }) => (
+			<div ref={inputRef}>
```

Comment:
What's the inputRef purpose here?

Reply:
A ref is needed otherwise the calendar popup doesn't know where to anchor itself. I've seen the comments you made on Alessandro's PR about inputRef, so I can fix it so it doesn't use inputRef once his PR is merged

Reply:
I've fixed it in the chip stories. I think this is an okay use of it though - it's no different from what the baseCalendarPicker.component does.